builtin/providers/rancher/provider.go

```diff
@@ -71,7 +73,7 @@ func init() {
 
 		"secret_key": "API secret used to authenticate with the rancher server",
 
-		"api_url": "The URL to the rancher API",
+		"api_url": "The URL to the rancher API, must include version uri (ie. v1 or v2-beta)",
```

Comment:
I don't mind appending it in the backend end for compatibility, but I think the correct setting is to include the version string. I know we have the non-beta version of V2 is over the horizon.  In general we include the API version string in both Rancher Compose and Rancher CLI sort of the convention we go for.